### PR TITLE
add: update npm packages to run local server

### PR DESCRIPTION
**Pull request by hwilson2563**
#36 This is the issue I reported. Since the fix was fairly simple I have added this pr. 

If you wish to not commit this to the code base, may I can edit to the readme to include the node version required to make the current package-lock.json or possibly instructing the user to remove their package-lock.json upon build.

Great project by the way! 👍

### CODE REVIEW

**Comment by acrosman** [comment]
@hwilson2563 can you check your node version? From context it looks like you're running v13 which reached end-of-life last spring. v10, v12, and v14 are the only ones currently receiving support. It's very possible I'm using newer features of JS that aren't included in some of the older LTE node versions (certainly I'd expect problems with 10, not sure about 12 but seems likely I don't pay super careful attention).

Can you update the README edit to indicate v14 or later instead of 13, and rather than editing the package-lock file edit package.json to use [set the engine](https://docs.npmjs.com/files/package.json#engines) for greater than 14 as well?  I'll rebuild the package lock file accordingly (it causes merge problems when PRs include the package-lock file anyway).

**Comment by hwilson2563** [comment]
@acrosman sounds like a plan. I will update the read me and complete the set engine in the package 👍

File: ReadMe.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ ElectronForce is an Electron based Salesforce Org exploration tool using the [JS
 
 Right now, while the project is setup to build native applications on Mac, Windows, and Linux, I'm not maintaining builds. So it's a bit developer focused and you'll need to have [Node.js installed](https://nodejs.org/en/download/). Granted this is a largely an API interface so some technical ability is expected but longer-term not the ability to understand JavaScript. Anyway you don't need to know how to use Node just need to have the tools around.
 
+Recommended Node Version: <=14
```
Comment (hwilson2563) [inline ReadMe.md:9]:
```suggestion
Recommended Node Version: 14.0.0 or greater
```

**Comment by hwilson2563** [comment]
I hope these edits are as requested. Please feel free to let me know if I can do anything further to improve this pr.

**Comment by acrosman** [comment]
This looks great.  Thanks for catching the issue and helping me avoid others running into similar concerns.

**Pull request merged**